source/content/guides/integrated-composer.md

```diff
@@ -52,9 +56,50 @@ Integrated Composer is a Pantheon platform feature that extends Composer <Popove
 
 1. Commit `composer.json` and `composer.lock` and push.
 
-   - Pantheon will run Composer, generate build artifacts, and deploy it to your Dev or Multidev environment.
+   - Pantheon will run Composer, build artifacts, and deploy the changes to your Dev or Multidev environment. 
+
+</Tab>
+
+<Tab title="Test and Live Environments" id="test-live">
+
+1. Clone the Git repository from the Pantheon site's dashboard.
+
+1. Run `composer install`:
+
+   ```bash{promptUser: user}
+    composer install -- no-dev
+   ```
+
+1. Add a new dependency locally:
+
+   ```bash{promptUser: user}
+    composer require drupal/pkg-name
+   ```
+
+1. Commit `composer.json` and `composer.lock` and push.
+
+   - Pantheon will run Composer, build artifacts, and deploy the changes to your Test or Live environment. 
+
+
+</Tab>
+
+</TabList>
 
-### Add a Package from a Private Repository
+## Remove Individual Site Dependencies
+
+You can remove site dependencies if they are no longer needed.
```

Comment:
These instructions are very dangerous. It is technically correct, but only under a very strict definition of "no longer needed". If a customer decides they "don't need" a module, but leave it installed and then follows these instructions, they can cause problems with their site. To avoid this, we should add a new first step to ensure that customers don't get into this situation.
```
1. Ensure that all modules in the package have been uninstalled. Uninstall them in the Drupal admin dashboard, or do it from the command line with Terminus:
    ```bash
    terminus drush site.live -- pm:uninstall module1 module2
   ```
Clone the database from live to all other environments before continuing.